locales/de.json

```diff
@@ -0,0 +1,623 @@
+      "incomplete": {
+        "bgContrast": "Das Kontrastverhältnis des Elements konnte nicht bestimmt werden. Suchen Sie nach einem bestimmten Hover/Fokus-Stil.",
+        "bgImage": "Das Kontrastverhältnis des Elements konnte aufgrund eines Hintergrundbildes nicht bestimmt werden.",
+        "bgGradient": "Das Kontrastverhältnis des Elements konnte aufgrund eines Hintergrundfarbverlaufes nicht bestimmt werden.",
+        "imgNode": "Das Kontrastverhältnis des Elements konnte nicht bestimmt werden, da das Element einen Image Node enthält.",
+        "bgOverlap": "Das Kontrastverhältnis des Elements konnte aufgrund einer Überlagerung nicht bestimmt werden.",
+        "default": "Das Kontrastverhältnis konnte nicht ermittelt werden."
+      }
+    },
+    "fieldset": {
+      "pass": "",
+      "fail": "{{var code = it.data && it.data.failureCode;}}{{? code === 'no-legend'}}Das Fieldset besitzt kein <legend>-Element als erstes Kindelement{{?? code === 'empty-legend'}}Das <legend>-Element enthält keinen Text, der für Screenreader sichtbar ist{{?? code === 'mixed-inputs'}}Das Fieldset besitzt unerlaubte Inputs{{?? code === 'no-group-label'}}Die ARIA-Gruppe besitzt kein aria-label- oder aria-labelledby-Attribut{{?? code === 'group-mixed-inputs'}}Die ARIA-Gruppe besitzt unerlaubte Inputs{{??}}Das Element besitzt kein(e) fieldset oder ARIA-Gruppe {{?}}"
+    },
+    "group-labelledby": {
+      "pass": "",
+      "fail": "Die Elemente mit dem Namen \"{{=it.data.name}}\" referenzieren mit aria-labelledby nicht alle dasselbe Element."
+    },
+    "accesskeys": {
+      "pass": "",
+      "fail": "Das Dokument enthält mehrere Elemente mit demselben accesskey-Attribut."
+    },
+    "focusable-no-name": {
+      "pass": "",
+      "fail": "Das Element befindet sich in der Tabreihenfolge und enthält keinen erreichbaren Text."
+    },
+    "landmark-is-top-level": {
+      "pass": "",
+      "fail": "Die {{=it.data.role }} landmark befindet sich innerhalb einer anderen landmark."
+    },
+    "page-has-heading-one": {
+      "pass": "",
+      "fail": "Die Seite muss eine Überschrift erster Ebene besitzen."
+    },
+    "page-no-duplicate-banner": {
+      "pass": "",
+      "fail": "Das Dokument besitzt mehr als eine banner landmark."
+    },
+    "page-no-duplicate-contentinfo": {
+      "pass": "",
+      "fail": "Das Dokument besitzt mehr als eine contentinfo landmark."
+    },
+    "page-no-duplicate-main": {
+      "pass": "",
+      "fail": "Das Dokument besitzt mehr als eine main landmark."
+    },
+    "tabindex": {
+      "pass": "",
+      "fail": "Das Element besitzt einen Tab-Index größer als 0."
+    },
+    "duplicate-img-label": {
+      "pass": "",
+      "fail": "Das Element besitzt ein <img>-Element mit Alternativtext, der vorhandenen Text wiederholt."
+    },
+    "explicit-label": {
+      "pass": "",
+      "fail": "Das Formular-Element besitzt kein explizites <label>."
```

Comment:
Instead of `Formular-Element`, `<form>-Element` for consistency?

Reply:
Sure! :-)